models/intel/handwritten-english-recognition-0001/accuracy-check.yml

```diff
@@ -0,0 +1,33 @@
+models:
+  - name: handwritten-english-recognition-0001
+
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: ctc_greedy_search_decoder
+          logits_output: output
+
+    datasets:
+      - name: GNHK
+        # In order to be used by model, images must be:
+        # 1) Resized to fixed height 96, using AREA interpolation
+        # 2) Padded with edge padding, right padding only
+        preprocessing:
+          - type: bgr_to_gray
+          - type: resize
+            interpolation: AREA
+            aspect_ratio_scale: width
+            size: 96
+          - type: padding
+            use_numpy: True
+            numpy_pad_mode: edge
+            dst_height: 96
+            dst_width: 2000
+            pad_type: right_bottom
+        #  - type: normalization
+        #    mean: 127.5
+        #    std: 127.5
+
+        metrics:
+          - type: label_level_recognition_accuracy
+
```

Comment:
please provide metric reference in accuracy config

Reply:
Reference metric added.